src/skin/socialwidgets/socialwidgets.css

```diff
@@ -42,14 +42,7 @@
  * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
  * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */
-
-.privacyBadgerReplacementButton {
-	border: none !important;
-	cursor: pointer !important;
-	height: auto !important;
-	width: auto !important;
-}
-
+	
 .privacyBadgerOriginalButton {
```

Comment:
Could we inline these as well? Then we can drop the whole `socialwidgets.css` file and remove a bunch of code from `socialwidgets.js` (yay!).

Reply:
I don't see why not - I'll check it out and follow up.

Reply:
I removed `socialwidgets.css` and relevant code from `socialwidgets.js`. This might warrant a quick review, but I don't think that I left any artifacts that added the stylesheet.